Userland/Libraries/LibJS/Runtime/Temporal/AbstractOperations.cpp

```diff
@@ -1052,33 +1053,32 @@ BigInt* round_number_to_increment(GlobalObject& global_object, BigInt const& x,
 }
 
 // 13.34 ParseISODateTime ( isoString ), https://tc39.es/proposal-temporal/#sec-temporal-parseisodatetime
-ThrowCompletionOr<ISODateTime> parse_iso_date_time(GlobalObject& global_object, [[maybe_unused]] String const& iso_string)
+ThrowCompletionOr<ISODateTime> parse_iso_date_time(GlobalObject& global_object, ParseResult const& parse_result)
```

Comment:
Who will do the parsing then...?

Reply:
Check out the subsequent commits, then use of this function should become more clear :^)
Basically, it cannot do parsing on its own as it only accesses productions parsed as children of other productions `TemporalFooString`, so not passing in the actual string is fine.